Guide_for_Contributors_New_to_Github.md

```diff
@@ -0,0 +1,124 @@
+
+    <img style="float: left;" src="https://github.com/Weimankow/tanzu-dev-portal/blob/Guide-for-Contributors-New-to-Github/static/images/guides/guide-for-contributors-new-to-github/Fork2.png" />
+1. Check the title of the repository to make sure that you are editing on your own copy of the code. It should show your github account name/tanzu-dev-portal, as you can see below:
+
+    <img style="float: left;" src="https://github.com/Weimankow/tanzu-dev-portal/blob/Guide-for-Contributors-New-to-Github/static/images/guides/guide-for-contributors-new-to-github/Fork3.png" />
+
+
+1. Click on Content
+
+    <img style="float: left;" src="https://github.com/Weimankow/tanzu-dev-portal/blob/Guide-for-Contributors-New-to-Github/static/images/guides/guide-for-contributors-new-to-github/Fork4.png" />
+1. Click on Practices. You’ll see a list of all the practices currently on the site.
+    <img style="float: left;" src="https://github.com/Weimankow/tanzu-dev-portal/blob/Guide-for-Contributors-New-to-Github/static/images/guides/guide-for-contributors-new-to-github/Fork5.png" />
+1. Click on the practice you want to add to. For this example, I clicked on design-studio.
+    <img style="float: left;" src="https://github.com/Weimankow/tanzu-dev-portal/blob/Guide-for-Contributors-New-to-Github/static/images/guides/guide-for-contributors-new-to-github/Fork6.png" />
+1. Click on index.md. This shows you what the page looks like in plain text. 
+
+1. Click on the pencil edit icon. This shows you what the page looks like in markdown language. Don’t worry, markdown is a simplified HTML/CSS script.   
+
+    <img style="float: left;" src="https://github.com/Weimankow/tanzu-dev-portal/blob/Guide-for-Contributors-New-to-Github/static/images/guides/guide-for-contributors-new-to-github/Fork8.png" />. 
+
+## 3. Edit (Make Edits and save as branch)
+
+1. Paste your edits into the right section. Just note that if you’re adding a ‘header’, ‘callout’ or any form of special formatted text, take a look to see how headers have been written, so you can copy the syntax. 
+
+    <img style="float: left;" src="https://github.com/Weimankow/tanzu-dev-portal/blob/Guide-for-Contributors-New-to-Github/static/images/guides/guide-for-contributors-new-to-github/Edit1.png" />
+    For example, I added a tip in the example above.   
+
+    A tip is written like this:
+     {{%callout%}}
+    This is a tip
+     {{/%callout%}}. 
+
+    Try to also keep to the indentation format so that the text looks neat and readable.
+
+    Unfortunately, at this point in time, it is not possible to preview what it will look like on the actual website, but you can click on the Preview Changes tab to see if the edits show up in the right places. 
+
+
+1. Scroll down to the Commit Changes section, and fill in the title. It defaults to ‘Update index.md’, but delete that and give it a short summary of **what you did** in less that 50 words. Refer to the image below for an example.
```

Comment:
Writing a good commit message is a skill in and of itself. My favorite resource on the topic is: https://chris.beams.io/posts/git-commit/

I wonder if we can link to that?